Issue/3 prometheus stats

iperdomo opened the pull request.

iperdomo committed:
[#3] Enables prometheus event listener provider

iperdomo committed:
[#3] Make sure the events directory exists and is writeable

iperdomo committed:
[#3] Deployment configuration for prometheus metrics

- We expose the files using a side-car container

dlebrero commented:
We have to tell Prometheus to scrape the pod: https://github.com/akvo/akvo-flow-maps/blob/master/ci/test/flow-maps.yaml.template#L26

iperdomo committed:
[#3] Update metrics configuration

- Use a published `akvo/prometheus-filesystem-exporter` container
  to avoid copying the `CMD` settings
- Copy the providers .jar file to not depend on jitpack.io
- Change the deployment definition to use the published container

dlebrero commented on deployment.yaml:
```diff
@@ -71,9 +74,16 @@ spec:
           - name: cloudsql
             mountPath: /cloudsql
+      - name: keycloak-metrics
+        image: akvo/prometheus-filesystem-exporter:0.3.1
```
Where is this container created and published?

iperdomo replied:
public Docker registry
